docs/3_simulation_box.rst

```diff
@@ -231,21 +233,21 @@ The used keywords can be very easily found in the relative  `User Guide <http://
 - Individual blocks containing several specifications for the molecules which will figure in the box, such as their .xyz file and the number of molecules of each type that will be placed inside the box. In our case, as specified by the ``inside cube -80. -80. -80. 80.`` key, we will be placing the molecules inside a cube with minimum coordinates (x,y,z) = (-80,-80,-80) and maximum coordinates (80,80,80): in other words, we will fill a cube of side 160.0 Å with our molecules. We set the coordinates between -80 and 80 (instead of, for example, 0 to 160) because, as specified by the keywords ``center`` and ``fixed 0. 0. 0. 0. 0. 0.``, we want to place our NC model in the center of our box.
 
 Once our input is ready, we can simply run the following command: ``packmol < settings.inp``.
-Once the script has run, the ``box.xyz`` output containing the box will be inside the working directory. 
+Once the script has run, the ``box.xyz`` output containing the box will be inside the working directory.
 
 Generating the .psf file
 ------------------------
-We will now need to build the Protein Structure File (.psf) of our simulation box, containing the molecular-level information required to apply the force field to our system over the course of our MD trajectory (here is an `example <https://github.com/nlesc-nano/Tutorials/tree/build_qd/docs/_files/3.box_ordered.psf.zip>`__ of what ours looks like. You can take a look at this `website <https://www.ks.uiuc.edu/Training/Tutorials/namd/namd-tutorial-unix-html/node23.html>`__ to get an idea of its structure). Let's now take a peek at its first lines:
+We will now need to build the Protein Structure File (.psf) of our simulation box, containing the molecular-level information required to apply the force field to our system over the course of our MD trajectory (here is an :download:`example <https://github.com/nlesc-nano/Tutorials/build_qd/docs/_files/3.box_ordered.psf.zip>`__ of what ours looks like. You can take a look at this `website <https://www.ks.uiuc.edu/Training/Tutorials/namd/namd-tutorial-unix-html/node23.html>`__ to get an idea of its structure). Let's now take a peek at its first lines:
```

Comment:
```suggestion
We will now need to build the Protein Structure File (.psf) of our simulation box, containing the molecular-level information required to apply the force field to our system over the course of our MD trajectory (here is an :download:`example <https://github.com/nlesc-nano/Tutorials/build_qd/docs/_files/3.box_ordered.psf.zip>` of what ours looks like. You can take a look at this `website <https://www.ks.uiuc.edu/Training/Tutorials/namd/namd-tutorial-unix-html/node23.html>`__ to get an idea of its structure). Let's now take a peek at its first lines:
```

You don't need the `__` suffix when using a directive, `:download:` itself will take care of generating the hyperlinl.

Reply:
FYI, you can easily apply suggested commits by clicking the `Commit suggestion` button (or `Add suggestion to batch` if you want to apply multiple suggestions.